src/rich_text.ts

```diff
@@ -105,9 +105,11 @@ export const enrichText = (userinput: string): string => {
     }
   })
   userinput = userinput.replaceAll(
-    new RegExp(/(`{1,3})(.*?)(\1)|(https?:\/\/[a-zA-Z0-9\-_\[\]\?\#\:\&\$\+\*\%\/\.\=\@]+)/ig),
-    (match, _openingBackticks, _closingBackticks, _textInBackticks, url) => {
-      if (url == undefined) { return match }
+    // new RegExp(/(`{1,3})(.*?)(\1)|(https?:\/\/[a-zA-Z0-9-_[\]?#:&$+*%.=@]+)/ig),
+    /(`{1,3})(.*?)(\1)|(https?:\/\/[a-zA-Z0-9\-_[\]?#:&$+*%/.=@]+)/ig,
```

Comment:
did you test if this still works?

Reply:
no but i hope it will. you can run the project because of the errors :D

Reply:
Oh true. Lets get rid of the tsconfig update until it is fully fixed. Failing style pipeline is bad enough. But failing build is not okay. I remember something about a different amount of escapes or backslashes in // vs as RegExp string. Might not be an issue here but would feel safer if this was tested.

Reply:
i tested it, it seems like everything works as expected

Reply:
poggies